host/frontend/host-orchestrator/operator.go

```diff
@@ -0,0 +1,401 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+//     Unless required by applicable law or agreed to in writing, software
+//     distributed under the License is distributed on an "AS IS" BASIS,
+//     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+//     limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	pool := NewDevicePool()
+	polledSet := NewPolledSet()
+	config := InfraConfig{
+		Type: "config",
+		IceServers: []IceServer{
+			IceServer{URLs: []string{"stun:stun.l.google.com:19302"}},
+		},
+	}
+
+	setupDeviceEndpoint(pool, config)
+	r := setupServerRoutes(pool, polledSet, config)
+
+	http.Handle("/", r)
+	if err := http.ListenAndServe(":8080", nil); err != nil {
```

Comment:
Maybe worth taking a port other than 8080 if the package is going to always run this, as many demo http servers run at port 8080 or 8000.

Reply:
Done